internal/configtxgen/genesisconfig/config.go

```diff
@@ -23,6 +25,9 @@ import (
 const (
 	// The type key for etcd based RAFT consensus.
 	EtcdRaft = "etcdraft"
+
+	// The type key for BFT Smart consensus
+	SmartBFT = "BFT"
```

Comment:
can we name the key as specific smartbft

Reply:
yes good idea